tests/libvmi/factory.go

```diff
@@ -56,25 +56,27 @@ func NewCirros(opts ...Option) *kvirtv1.VirtualMachineInstance {
 
 // NewAlpine instantiates a new Alpine based VMI configuration
 func NewAlpine(opts ...Option) *kvirtv1.VirtualMachineInstance {
-	alpineMemory := cirrosMemory
-	alpineOpts := []Option{
-		WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
-		WithResourceMemory(alpineMemory()),
-		WithRng(),
-	}
+	alpineOpts := []Option{WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskAlpine))}
+	alpineOpts = append(alpineOpts, AlpineOptions()...)
 	alpineOpts = append(alpineOpts, opts...)
 	return New(alpineOpts...)
 }
 
 func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	alpineOpts := []Option{WithContainerImage(cd.ContainerDiskFor(cd.ContainerDiskAlpineTestTooling))}
+	alpineOpts = append(alpineOpts, AlpineOptions()...)
+	alpineOpts = append(alpineOpts, opts...)
+
+	return New(alpineOpts...)
+}
+
+func AlpineOptions() []Option {
```

Comment:
I do not understand why this is needed. It also looks odd, as these options have nothing to do with Alpine really.

Reply:
this are default Alpine options that you can use when creating alpine VMs with some PVCs, DVs and other disks, this can also be reused in the factory functions